docs/api/wanchain.md

```diff
@@ -563,6 +568,14 @@ A class instance `variable` holding a `string` which represents a Wanchain accou
 When you are deploying a new ledger, this variable represents the ledger ID and is `null` until a mutation is completed.
```

Comment:
When you are deploying a new ledger, this variable represents the ledger ID and remains `null` until a mutation is completed.